.travis.yml

```diff
@@ -57,6 +57,18 @@ script:
 - ./test.sh
 - if [ "$TRAVIS_OS_NAME" = "linux" ]; then PATH=$PATH:$(pwd) ./test_e2e.sh; fi
 
+jobs:
+  include:
+    - stage: Go Coverage
+      before_script:
+        - go get github.com/mattn/goveralls
+      script:
+        - make test-coverage
+        # todo(camilamacedo86): Add env var with repotoken value in trevis. It should not be a public info. Then, replace as follows
+        # todo(camilamacedo86): ReGen the token in https://coveralls.io/github/kubernetes-sigs/kubebuilder
+        # - $GOPATH/bin/goveralls -service=travis-ci -coverprofile=coverage-all.out -repotoken=$COVERALLS_TOKEN
```

Comment:
If you need help from us with the token, please let me know.

Reply:
It is not a good practice to leave the token public. So, the best is to create an env var and add it. To do it, just go in the Trevis > project > more options > settings >> add env var.

<img width="204" alt="Screenshot 2019-11-26 at 09 11 19" src="https://user-images.githubusercontent.com/7708031/69615531-f39d5000-102c-11ea-9127-12ac533f61df.png">
<img width="1045" alt="Screenshot 2019-11-26 at 09 11 31" src="https://user-images.githubusercontent.com/7708031/69615537-f5ffaa00-102c-11ea-97d5-f824939c2f3c.png">

Reply:
Hi @mengqiy and @alexeldeib,
IHMO it is not a blocker to merge this one. However, would be great to have a pull request in a sequence of it to add the commented line and remove this one in order to use the token stored as env var of Travis.

Reply:
I've regenerated the token and added it as the `COVERALLS_TOKEN` env var in travis.  Please update accordingly.

Reply:
Done @DirectXMan12 really tks :-)